scripts/install_backend_python_libs.py

```diff
@@ -28,23 +28,44 @@
 import pkg_resources
 
 
-def _normalize_string_that_contains_python_library_name(original_string):
-    """Python library name strings are case insensitive which means that
+def _normalize_python_library_name(library_name):
+    """Returns a normalized (lowercase) version of the python library name.
+
+    Python library name strings are case insensitive which means that
     libraries are equivalent even if the casing of the library names are
-    different. This function normalizes all strings containing python library
-    names so that strings with identical libraries have identical library name
-    strings.
+    different. This function normalizes python library names such that
+    identical libraries have the same library name strings.
+
+    Args:
+        library_name: str. The library name to be normalized.
+
+    Returns:
+        str. A normalized library name that is all lowercase.
+    """
+    return library_name.lower()
 
-    This is for comparing library names with different case and directory names
-    generated from installing libraries with different case.
+def _normalize_directory_name(directory_name):
+    """Returns a normalized (lowercase) version of the directory name.
+
+    Python library name strings are case insensitive which means that
+    libraries are equivalent even if the casing of the library names are
+    different. When python libraries are installed, the generated metadata
+    directories also use the python library names as part of the directory name.
+    This function normalizes directory names so that metadata directories with
+    different case won't be treated as different in code. For example,
+    `GoogleAppEnginePipeline-1.9.22.1.dist-info` and
+    `googleappenginepipeline-1.9.22.1.dist-info` are equivalent, generated by
+    the same python installation but their names are not the same. To make sure
```

Comment:
Maybe "generated by the same python installation but" --> "although"

(Why would the same python installation generate two different dirs?)